### ISSUE TITLE
Upgrade upgrades all towers

### ISSUE THREAD

**Issue by john9francis** [john9francis/sushi-attack#7]
Instead of having my upgrade node, I should just impliment an "upgrade" function in each of the towers. Then instead of calling the "upgrades" group we can call the "CsTowers" group

but wait, that wouldn't work either, that would upgrade all the cstowers

The next idea would be using the towerPlatform, checking the children, and if the children are in the group "towers," call the upgrade function. 

Basically call the function on the tower-platform's child.

**Comment by john9francis** [comment]
I changed upgrade functionality so clicking the upgrade button will upgrade only the one tower. Also all towers can now be upgraded twice.
